### PR TITLE
Fix/#1994 secure dwr methods 2 7 1

### PR DESCRIPTION
**Pull request by Limraj**

### CODE REVIEW

File: src/com/serotonin/mango/web/dwr/DataSourceListDwr.java [hunk 1]
```diff
@@ -112,7 +114,7 @@ public int compare(Type o1, Type o2) {
 	}
 
 	public int deleteDataSource(int dataSourceId) {
-		Permissions.ensureDataSourcePermission(Common.getUser(), dataSourceId);
+		Permissions.ensureAdmin();
```
Comment (radek2s) [inline src/com/serotonin/mango/web/dwr/DataSourceListDwr.java:117]: Using "ensureAdmin()" we are limiting the usage of this "deleteDataSource" method only to administrators. Are we sure that we do not want to permit users with DataSource permission also to delete a DataSource?

File: src/com/serotonin/mango/web/mvc/controller/DataPointDetailsController.java [hunk 2]
```diff
@@ -50,7 +50,7 @@ protected ModelAndView handleRequestInternal(HttpServletRequest request,
 			HttpServletResponse response) throws Exception {
 		Map<String, Object> model = new HashMap<String, Object>();
 		User user = Common.getUser(request);
-
+		Permissions.ensureAdmin(user);
```
Comment (radek2s) [inline src/com/serotonin/mango/web/mvc/controller/DataPointDetailsController.java:53]: Why we are blocking the "data point details" page?  
If user has at least "read" access to specific data point I think that he should have right to open this page.

File: src/org/scada_lts/web/mvc/controller/ScriptController.java [hunk 3]
```diff
@@ -59,6 +60,7 @@
 		// TODO ta metoda powinna przyjmowac jako argument tresc skryptu z
 		// odpowiednimi argumentami nie powinno to isc przez baze danych !!!.
 		User user = Common.getUser(request);
+		Permissions.ensureAdmin(user);
```
Comment (radek2s) [inline src/org/scada_lts/web/mvc/controller/ScriptController.java:63]: The same issue is here. Are we sure that we want to disable scriptExecution invoked by non-admin-user in ScriptController? I know that we do not have "script objects permission" but what if someone who is not an admin wants to execute existing script? That is also a business logic change.

File: src/com/serotonin/mango/web/dwr/DataSourceListDwr.java (hunk 1 above)
Comment (Limraj) [reply src/com/serotonin/mango/web/dwr/DataSourceListDwr.java:117]: No user except admin can add / remove datasource.

File: src/com/serotonin/mango/web/mvc/controller/DataPointDetailsController.java (hunk 2 above)
Comment (Limraj) [reply src/com/serotonin/mango/web/mvc/controller/DataPointDetailsController.java:53]: The read permission is applied to visible on view, and you can go to history, loupe on watchlist.

File: src/org/scada_lts/web/mvc/controller/ScriptController.java (hunk 3 above)
Comment (Limraj) [reply src/org/scada_lts/web/mvc/controller/ScriptController.java:63]: You may be right, I'll check tomorrow.

**Review by radek2s — APPROVED** [review]

**Pull request merged**